app/src/ui/diff/diff-options.tsx

```diff
@@ -71,12 +71,23 @@ export class DiffOptions extends React.Component<
     })
   }
 
-  private onHideWhitespaceChangesChanged = (
+  public componentWillUnmount() {
+    document.removeEventListener('mousedown', this.onDocumentMouseDown)
+  }
+
+  private onDocumentMouseDown = (event: MouseEvent) => {
+    const { current: ref } = this.diffOptionsRef
+    const { target } = event
+
+    if (ref !== null && target instanceof Node && !ref.contains(target)) {
+      this.closePopover()
+    }
+  }
```

Comment:
Is this a leftover after rebasing? In theory I got rid of this in a [previous refactor](https://github.com/desktop/desktop/pull/11571/files#diff-2c0fe2274b8c57c0fbfa6c5763627f504e43bf4e59526f131582b456d517197aL84) 😳